### PR TITLE
QOLOE-171 Typography QA

### PR DESCRIPTION
**Pull request by tedqgov**
QA issue: https://ssq-qol.atlassian.net/browse/QOLOE-171

### CODE REVIEW

File: src/scss/qld-type.scss [hunk 1]
```diff
@@ -55,7 +55,7 @@ $link-visited-color:          $qld-link-visited;
     --qld-link-hover-color: #FFF;
     --qld-link-hover-color-rgb: 255, 255, 255;
 
-    --qld-link-visited: var(--qld-link-visited-dark);
+    --qld-link-visited: var(--qld-link-visited-dark)!important;
```
Comment (duttonw) [inline src/scss/qld-type.scss:58]: is !important really required?
There should be a better way of doing this.

File: src/scss/qld-type.scss [hunk 2]
```diff
@@ -70,6 +70,21 @@ body {
...
+ul, ol, p, .accordion-body {
+    line-height: 1.75;
+}
+
+li {
+    margin-top: 0.5rem;
+}
+
+.nav, .nav ul {
+    line-height: 1.5!important;
```
Comment (duttonw) [inline src/scss/qld-type.scss:82]: ditto (is there a better way instead of getting out the override hammer)

File: src/scss/qld-type.scss [hunk 3]
```diff
@@ -84,6 +99,17 @@ body {
     h6 {
         color: var(--qld-headings-color);
     }
+        // .dark, .dark-alt & {
+        a, a.nav-link {
+            text-decoration-color: #B5CCE2;
+            &:visited {
+                color: var(--qld-link-visited-dark)!important;
```
Comment (duttonw) [inline src/scss/qld-type.scss:106]: simliar

**Review by duttonw — CHANGES_REQUESTED** [review]
Can another help out in showing a way to not need !important. It should only be used sparingly and this is clean room, so you should be able to rejig the css to accommodate what you want to do. 

Leave this in will cause much pain in the future.

**Comment by re-becca** [comment]
@tedqgov Noticed two minor things for link underline:

- light/alternative - visited:hover should have a text-decoration-color of #551A8B (use associated variable)
- dark/darkalt - default:hover should have a text-decoration-color of #FFFFFF (use associated variable)

**Comment by tedqgov** [comment]
@re-becca Alright, I made the changes and cleaned up the code more, also added a new colour variable "$qld-link-dark-underline" in the variables file -- let me know if there's anything else you need :)

**Review by MitchAnderson93 — CHANGES_REQUESTED** [review]

File: src/scss/qld-variables.scss [hunk 4]
```diff
@@ -28,8 +28,9 @@ $qld-light-background: #EFF4F9;
 $qld-light-background-shade: #E5EEF5;
 $qld-light-border: #ECEDE9;
 $qld-dark-focus: #01B0E5;
-$qld-link-visited: #8B63B0;
-$qld-link-visited-dark: #E1C2FF;
+$qld-link-visited: #551A8B;
```
[the remark this answers is not in the document]
Comment (tedqgov) [reply src/scss/qld-variables.scss]: @MitchAnderson93 Yeah I noticed when fixing up conflicts, I left the colour in hex code for now in qld-type.scss

**Comment by tedqgov** [comment]
Had to reset my commit due to merge conflicts, will take a while

**Comment by duttonw** [comment]
Hi @tedqgov ,
Is this included in https://github.com/qld-gov-au/qgds-bootstrap5/pull/214


So this can be closed then?

**Comment by tedqgov** [comment]
@duttonw Nope not included, will delete this branch -- I cherrypicked commits to the new branch due to git rebase looping bug.

**Pull request closed without merging**